cypress/e2e/signIn.cy.js

```diff
@@ -12,8 +14,8 @@ describe('Sign In page', () => {
 
   before(() => {
     cy.task('db:clear');
```

Comment:
The `before` hook is clearing the database. Ensure this is necessary for setting up the test environment. If the database state does not need to be reset for these tests, consider removing this operation to optimize test performance.